template/apps/reactive-resume.json

```diff
@@ -4,22 +4,61 @@
 		"Tools"
 	],
 	"description": "A one-of-a-kind resume builder that's not out to get your data. Completely secure, customizable, portable, open-source and free forever.",
-	"image64": "martadinata666/reactive-resume:latest",
 	"logo": "https://raw.githubusercontent.com/pi-hosted/pi-hosted/master/images/reactiveresume.png",
 	"name": "reactive-resume",
 	"officialDoc": "https://hub.docker.com/r/martadinata666/reactive-resume",
+	"preInstallScript": "install_reactive-resume.sh",
 	"platform": "linux",
-	"ports": [
-		"7768:80/tcp"
-	],
-	"restart_policy": "unless-stopped",
-	"title": "Reactive-Resume",
-	"type": 1,
-	"volumes": [
+  "env": [
+		{
+			"default": "some-random-alphanumeric123456",
+			"label": "SECRET_KEY",
+			"name": "SECRET_KEY"
+		},
 		{
-			"bind": "/portainer/Files/AppData/Config/ReactiveResume",
-			"container": "/keepoutoftheway/var/www/html"
+			"default": "some-random-alphanumeric123456",
+			"label": "JWT_SECRET",
+			"name": "JWT_SECRET"
+		},
+                {
+			"default": "reactiveresume",
+			"label": "DATABASE_USERNAME",
+			"name": "DATABASE_USERNAME"
+		},
+                {
+			"default": "reactiveresume",
+			"label": "DATABASE_PASSWORD",
+			"name": "DATABASE_PASSWORD"
+		},
+                {
+			"default": "api-key",
+			"label": "GOGLE_API_KEY",
+			"name": "GOOGLE_API_KEY"
+		},
+                {
+			"default": "false",
+			"label": "DISABLE_SIGNUPS",
+			"name": "DISABLE_SIGNUPS"
+		},
+                {
+			"default": "7768",
+			"label": "PORT",
+			"name": "PORT"
+		},
+		{
+			"default": "America/New_York",
+			"label": "TZ",
+			"name": "TZ"
 		}
```

Comment:
The reason i don't include `ENABLE_S3_STORAGE` is because S3 storage itself not really work, "minio" dont work. I never tried AWS/Digital Ocean S3 storage then by giving user to enable it making it hard to support. So i think hide `ENABLE_S3_STORAGE` viable options and keep it simple and local.

Reply:
It does makes sense, but then you need to remove the variable from the stack file

Reply:
Is just set `      - STORAGE_S3_ENABLED=false` will do?